codegen/build-oas.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -eux -o pipefail
+
+version=$1 # e.g. 2024-07
+modules=("control")
```

Comment:
Unlike python and ts, since Java SDK is gRPC only for data plane, is that the reason we have control only mentioned in the modules?

Reply:
Yes, we don't need the rest actions for java. But I still left this basic structure in place because I think in the future we could easily end up with multiple things here. For example, if we eventually expose management plane operations.